source/basic.tex

```diff
@@ -5400,6 +5579,94 @@
 pointer-to-member conversions\iref{conv.ptr,conv.mem}.
 \tcode{\keyword{sizeof}(std::nullptr_t)} shall be equal to \tcode{\keyword{sizeof}(\keyword{void}*)}.
 
+\pnum
+A value of type \tcode{std::meta::info} is called a \defn{reflection}.
+There exists a unique null reflection;
```

Comment:
In P2996R13 this is a definition of 'null reflection'

Reply:
fixed